contracts/Ibc.sol

```diff
@@ -45,3 +58,10 @@ struct PacketFee {
     uint256 ackFee;
     uint256 timeoutFee;
 }
+
+struct Proof {
+    // block height at which the proof is valid for a membership or non-membership at the given keyPath
+    uint64 proofHeight;
```

Comment:
This is ok for now.   (As we use this single integer `height` for polymer height)

- We have various places use integer height. 
- This means, if we do `polymer` upgrade, it will be hard to 
   increase `polymer revision_number`  (hard fork). 

We will see if we need that revision_number bump when we do first staging chain upgrade.   

If we do need revision_number, then we need to improve multiple components to support that (revision, height) pair.

Reply:
To be future proof, I changed this to `Height { reversion_number, revision_height } `